mysql/Doc_Office.sql

```diff
@@ -1,87 +1,182 @@
 DROP DATABASE IF EXISTS Doc_Office;
 CREATE DATABASE Doc_Office;
 USE Doc_Office;
-DROP TABLE IF EXISTS DOCTOR;
 
 DROP TABLE IF EXISTS ACCOUNT;
 CREATE TABLE ACCOUNT(
 	username varchar(45) NOT NULL UNIQUE,
     role varchar(45),
     hash varchar(100)
     );
+    
+DROP TABLE IF EXISTS DOCTOR;
 CREATE TABLE DOCTOR(
-	  Doctor_ID int NOT NULL UNIQUE AUTO_INCREMENT,
-    First_name varchar(45) ,
+	Doctor_ID int NOT NULL UNIQUE AUTO_INCREMENT,
+    First_name varchar(45),
     Last_name varchar(45),
-    Specialty varchar(45),
-    Salary int,
     username varchar(45),
+    Salary int,
     Phone_number char(10),
-    FOREIGN KEY (username) REFERENCES ACCOUNT(username),
-    Primary Key (Doctor_ID)
+    PRIMARY KEY (Doctor_ID),
+    FOREIGN KEY (username) REFERENCES ACCOUNT(username)
+    );
+    
+DROP TABLE IF EXISTS APPOINTMENT;
+CREATE TABLE APPOINTMENT(
+	Appointment_number int AUTO_INCREMENT,
+    date datetime,
+    doctorid int,
+    roomNumber int,
```

Comment:
Don't switch between different naming schemes, either choose camel case or your way of doing it, doctorid and roomNumber and Appointment_number are all different